install/kubernetes/cilium/templates/cilium-resource-quota.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if and (ne .Release.Namespace "kube-system" ) .Values.gke.enabled }}
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: cilium-resource-quota
+  namespace: {{ .Release.Namespace }}
+spec:
+  hard:
+    # 5k nodes * 2 DaemonSets (Cilium and cilium node init)
+    pods: "10k"
```

Comment:
Just out of curiosity, where can I find more details about this number - 5k nodes from scalability perspective?

Reply:
@aditighag https://kubernetes.io/docs/setup/best-practices/cluster-large/#support

Reply:
Surely we should allow users to customize this via Helm?